Add the new MPM to the build system as an alternative to the other MPMs,
and mark it as experimental.

Index: httpd-2.2.17/server/mpm/config.m4
===================================================================
--- httpd-2.2.17.orig/server/mpm/config.m4
+++ httpd-2.2.17/server/mpm/config.m4
@@ -1,7 +1,7 @@
 AC_MSG_CHECKING(which MPM to use)
 AC_ARG_WITH(mpm,
 APACHE_HELP_STRING(--with-mpm=MPM,Choose the process model for Apache to use.
-                          MPM={beos|event|worker|prefork|mpmt_os2|winnt}),[
+                          MPM={beos|event|worker|prefork|mpmt_os2|winnt|itk}),[
   APACHE_MPM=$withval
 ],[
   if test "x$APACHE_MPM" = "x"; then
@@ -23,7 +23,7 @@ ap_mpm_is_threaded ()
 
 ap_mpm_is_experimental ()
 {
-    if test "$apache_cv_mpm" = "event" ; then
+    if test "$apache_cv_mpm" = "event" -o "$apache_cv_mpm" = "itk" ; then
         return 0
     else
         return 1
